cmd/recorder.go

```diff
@@ -253,34 +254,45 @@ func cleanStaleFile(filename string) {
 	}
 }
 
+func transcode(inputFile, encoding, bitrate string) {
+	basename := filepath.Base(inputFile)
+	basenameWithoutExt := strings.TrimSuffix(basename, filepath.Ext(basename))
+	outputFile := fmt.Sprintf("%s/%s-%s-%%03d.m4s", MediaDir, basenameWithoutExt, bitrate)
+	cmd := exec.Command(
+		// Call ffmpeg on the most-recently created FLAC file
+		"ffmpeg", "-hide_banner", "-i", inputFile,
```

Comment:
what language is this? certainly not go.

Reply:
Correct, I'm using [ffmpeg](https://www.ffmpeg.org/) here to convert the raw FLAC files written to disk into HLS-compliant formats